frontend/src/app/modules/members/members-page/members-page.component.html

```diff
@@ -82,12 +82,13 @@
 
                     <div class="content" *ngIf="memberItem.isEdit">
                         <p-dropdown
-                            [(ngModel)]="memberItem.member.role" 
+                            [(ngModel)]="memberItem.member.role"
                             [options]="roles"
                             appendTo="body"
+                            [disabled]="memberItem.saveRole.name === 'Owner'"
                             optionLabel="name" name="roles-dropdown"
                             placeholder="Select Role" [filter]="false"></p-dropdown>
-                        <button (click)="saveRoleChanges(memberItem)"
+                        <button (click)="saveRoleChanges(memberItem)" [disabled]="memberItem.saveRole.name === 'Owner'"
```

Comment:
It would be better to not disable this button for the owner. An owner should be able to change a role for all members